nixos/modules/services/security/esdm.nix

```diff
@@ -0,0 +1,137 @@
+{ lib, config, pkgs, ... }:
+
+let
+  cfg = config.services.esdm;
+  kernelVersion = config.boot.kernelPackages.kernel.version;
+  linux_6_3 = (lib.versionAtLeast kernelVersion "6.3");
+in
+{
+  options.services.esdm = {
+    enable = lib.mkEnableOption (lib.mdDoc "ESDM service configuration");
+    package = lib.mkPackageOptionMD pkgs "esdm" { };
+    kernelEnable = lib.mkOption {
```

Comment:
This should also use mkEnableOption

Reply:
`kernelEnable` is now changed from mkOption to mkEnableOption. It is also renamed now to give more information on what it does. And the description should now be more informative.
(See commit: https://github.com/NixOS/nixpkgs/pull/238777/commits/7dab9541778a79c20bc135cfc2a8fcafa2adcaf5)